Modules/Filtering/Smoothing/test/itkDiscreteGaussianImageFilterTest2.cxx

```diff
@@ -84,23 +86,33 @@ itkDiscreteGaussianImageFilterTest2(int argc, char * argv[])
     return EXIT_FAILURE;
   }
 
-  float        sigma = (argc >= 6) ? std::stof(argv[5]) : 0.0;
-  float        kernelError = (argc >= 7) ? std::stof(argv[6]) : 0.01;
-  unsigned int kernelWidth = (argc >= 8) ? static_cast<unsigned int>(std::stoi(argv[7])) : 32;
+  float        sigma = (argc > 5) ? std::stof(argv[5]) : 0.0;
```

Comment:
Changes which are not related to increase of test coverage could be squashed into the previous commit.